CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -406,6 +406,9 @@ strFacilityGTSBody=The Guerilla Tactics School has undergone an overhaul. It now
 strActivityChainsHeader=Activity Chains
 strActivityChainsBody=An activity chain is a set of missions and/or covert actions that yield big rewards. They will be presented to you by the resistance, and as the communication network grows, more opportunities will be revealed. While the main reward(s) are usually given at the end of the chain, the earlier missions are still beneficial for XP and loot and often include secondary rewards. For more info, click the View Chain button at the top of the screen.
 
+strResistanceInformantsHeader=Resistance Informants
+strResistanceInformantsBody="- In the top right corner of this screen, you can see the current number of Informants. These resistance sympathizers, hidden in ADVENT controlled areas, are the key to XCOM's global operations.<br/>- Informants gather data and report back to XCOM, revealing new opportunities for our soldiers to investigate. With our current number, we can expect our informants to uncover a new activity chain about every two weeks.<br/>- The ADVENT administration in this region is already completely penetrated. Expanding the Resistance Network by contacting new regions or building radio relays will increase the number of resistance Informants at our disposal.<br/>- A high priority for XCOM should be to extend the Resistance Network as soon as possible."

```

Comment:
> The ADVENT administration in this region is already completely penetrated. Expanding the Resistance Network by contacting [...]

These 2 sentences are really disconnected which makes it confusing to read. Change it to something like "The ADVENT administration in this region is already completely penetrated, so if we want to find out more, we should expand the resistance network to other regions"

---------------

Why did you use `-` instead of `<Bullet/>`?

Reply:
For some reason, the `<Bullet/>`s were invisible